packages/blade/src/components/Dropdown/docs/DropdownWithButton.stories.tsx

```diff
@@ -161,6 +161,57 @@ InternalMenu.parameters = {
   },
 };
 
+export const InternalAutoPositioning = (): React.ReactElement => {
```

Comment:
why mark this as Internal? this is a good to have for consumer's ref too?

Reply:
event these
<img width="274" alt="Screenshot 2023-08-25 at 11 03 35 AM" src="https://github.com/razorpay/blade/assets/11384858/83499aaf-2c93-4bad-8e89-f8cf0c184dec">

Reply:
We already have two examples for auto positioning 
- https://blade.razorpay.com/?path=/story/components-dropdown-with-select--with-bottom-aligned-select
- https://blade.razorpay.com/?path=/story/components-dropdown-with-button-and-link--with-right-aligned-menu


I just needed one example where I can test locally and see all positions at one place since this is important scenario for Dropdown